### PR TITLE
Verify that network IDs match

### PR DESCRIPTION
**Pull request by tinybike**
[Clubhouse Story](https://app.clubhouse.io/augur/story/6692)

## Submitter checklist
- [x] Test covering functionality added/fixed in
- Check the linked story includes the following:
  - [x] Steps to reproduce
  - [ ] Screenshot (If applicable)

## Reviewer Checklist
- [ ] Test/lint pass 
- [ ] Post merge, story marked complete

### CODE REVIEW

File: src/modules/app/actions/init-augur.js [hunk 1]
```diff
@@ -35,51 +42,31 @@ function pollForAccount(dispatch, getState) {
...
-            default:
-              return expectedNetwork
-          }
-        }
-
+    const { modal } = getState()
+    verifyMatchingNetworkIds((err, isMatchingNetworkIds) => {
+      if (err) return console.error('pollForNetwork failed', err)
+      if (isMatchingNetworkIds === false && isEmpty(modal)) {
+        const expectedNetworkId = AugurJS.augur.rpc.getNetworkID()
```
Comment (justinbarry) [inline src/modules/app/actions/init-augur.js]: This seems to be the network metamask is using. Should it be the network of the ether connected ether node?

screen shot 2018-03-07 at 11 23 41 pm

**Comment by tinybike** [comment]
@justinbarry Fixed/improved the network mismatch messaging a bit.

**Comment by tinybike** [comment]
Updating this so that it does not repeatedly request network ID from augur-node, as per conversation on Discord.

**Comment by justinbarry** [comment]
@tinybike The updated copy looks good. 

How do I test a using non-meta-mask network? I'm a bit of a neophyte so please forgive the questions of  this simpleton ;)

screen shot 2018-03-08 at 1 33 09 pm

**Comment by tinybike** [comment]
Ok @justinbarry, this should be good to go now!

**Pull request merged**